content/webapp/pages/search/index.tsx

```diff
@@ -262,31 +339,56 @@ export const SearchPage: NextPageWithLayout<Props> = ({
     try {
       const imagesResults = await getImages({
         params,
-        pageSize: 1,
+        pageSize: 5,
         toggles: data.toggles,
       });
       images = getQueryResults({
         categoryName: 'images',
         queryResults: imagesResults,
       });
-      setClientSideImages(images);
+      setClientSideImages({
+        totalResults: images?.totalResults || 0,
+        results:
+          images?.pageResults.map(image => ({
+            ...image,
+            src: image.locations[0].url,
+            width: (image.aspectRatio || 1) * 100,
+            height: 100,
+          })) || [],
+      });
     } catch (e) {
       return undefined;
     }
   }
-  useEffect(() => {
+
+  useSkipInitialEffect(() => {
```

Comment:
ohh I didn't know we had that! So it's to skip the initial one as `query` triggers a second one?

Reply:
yeah, I'm just double checking I do want to do that